examples/psa_crypto/example_ecdsa_p256.c

```diff
@@ -99,6 +102,10 @@ psa_status_t example_ecdsa_p256(void)
         return status;
     }
 
+#ifdef SECURE_ELEMENT
+    return psa_verify_hash(pubkey_id, ECC_ALG, hash, sizeof(hash), signature, sig_length);
```

Comment:
Would be great to have a comment here how this differs from `psa_verify_message()`

Reply:
I added a comment on why it's necessary to call `psa_verify_hash`.

Reply:
I'm just surprised because `psa_verify_message()` has no reference to a secure element, so I would expect it would work either way.

Reply:
I think a simple fallback for secure elements or other configurations where signing/verifying messages is not directly supported could be the following: internally calculate the hash using `psa_hash_compute()` (on the secure element, or in software if not supported) and then call `psa_verify_hash()` on the hash.

That's what I did for ed25519 on nRF52840dk iirc.

Reply:
I can do that, but I'd rather submit a separate PR for it, since it would be a new feature =)